ext/config.w32

```diff
@@ -0,0 +1,12 @@
+// vim:ft=javascript
+
+ARG_WITH("maxminddb", "Enable MaxMind DB Reader extension support", "no");
+
+if (PHP_MAXMINDDB == "yes") {
+	if (CHECK_HEADER_ADD_INCLUDE("maxminddb.h", "CFLAGS_MAXMINDDB", PHP_MAXMINDDB +  ";" + PHP_PHP_BUILD + "\\include\\maxminddb") &&
+		CHECK_LIB("libmaxminddb.lib", "maxminddb", PHP_MAXMINDDB)) {
+		EXTENSION("maxminddb", "maxminddb.c");
+	} else {
+		WARNING('Could not find maxminddb.h or maxminddb lib; skipping');
```

Comment:
Maybe "Could not find maxminddb.h or libmaxminddb.lib" to be clearer.

Reply:
Resolved by https://github.com/maxmind/MaxMind-DB-Reader-php/pull/115/commits/03c7b2f2ccc1555110f83bc077c462ac6ee463e3

Reply:
I forgot the dot: https://github.com/Jan-E/MaxMind-DB-Reader-php/commit/4bffde00dd1b96952c191d10a1de8dccbc7a33ac